working-groups.md

```diff
@@ -0,0 +1,169 @@
+# Tekton working group
+
+Most community activity is organized into _working groups_.
+
+Working groups follow the [contributing](./CONTRIBUTING.md) guidelines although
+each of these groups may operate a little differently depending on their needs
+and workflow.
+
+The working groups generate design docs which are kept in a
+[shared drive]()
```

Comment:
no link, that's normal?

Reply:
> I need to bootstrap some of the WG listed below :angel: and also, we should make sure I did put the right link at the right place :nerd_face: 

Waiting for that :angel:

Reply:
(that's why the `/hold` :upside_down_face:)

Reply:
you know that i never read descriptions 😝